manual.txt

```diff
@@ -0,0 +1,25 @@
+Pair number:  1  major color: White    minor_color: Blue
```

Comment:
this file is an output file if I am not wrong. Should this be also part of archive?